Dockerfile

```
@@ -1,5 +1,5 @@
-# Use the official Ruby 3.2.0 Alpine image as the base image
-FROM ruby:3.2.0-alpine
+# Use the official Ruby 3.3.5 Alpine image as the base image
+FROM ruby:3.3.5-alpine
```

Cool to float on 3.3.x. Can drop the self-explanatory comment as well.
```suggestion
FROM ruby:3.3-alpine
```

---

Done!